components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.jsx

```diff
@@ -146,6 +147,9 @@ export default class PermissionsTree extends React.Component {
         if (scope === 'team_scope' && this.groups[0].id !== 'teams_team_scope') {
             this.groups[0].id = 'teams_team_scope';
         }
+        if (license.LDAPGroups === 'true' && postsGroup.permissions.indexOf(Permissions.USE_GROUP_MENTIONS) === -1) {
```

Comment:
Update license check to match above suggested one? It seems that `postsGroup.permissions` is an array. Wouldn't it be better if we did `!postsGroup.permissions.inlcudes(Permissions.USE_GROUP_MENTIONS)` instead of using indexOf?

Reply:
yea i used `indexOf` initially since the other checks in this function also used it. But I decided to change all of them to use `includes` since its much more readable